src/lib/create-from-script-tag.js

```
@@ -0,0 +1,65 @@
+'use strict';
+
+var find = require('./find-parent-form');
+var uuid = require('./uuid');
+var DropinError = require('./dropin-error');
+
+function createFromScriptTag(createFunction, scriptTag) {
+  var authorization, containerId, container, form;
+
+  if (!scriptTag) {
+    return;
+  }
+
+  authorization = scriptTag.getAttribute('data-braintree-dropin-authorization');
+
+  if (!authorization) {
```

Do we want to throw an error if there is a script tag but no authorization?